internal/worker/modelworkermanager/modelworkermanager.go

```diff
@@ -265,7 +265,7 @@ func (m *modelWorkerManager) modelChanged(ctx context.Context, modelUUID string)
 
 func (m *modelWorkerManager) newWorkerFuncFromConfig(ctx context.Context, cfg NewModelConfig) (func(context.Context) (worker.Worker, error), error) {
 	modelUUID := model.UUID(cfg.ModelUUID)
-	modelName := fmt.Sprintf("%q (%s)", fmt.Sprintf("%s-%s", cfg.ModelOwner, cfg.ModelName), modelUUID)
+	modelName := fmt.Sprintf("%q (%s)", fmt.Sprintf("%s/%s", cfg.ModelQualifier, cfg.ModelName), modelUUID)
```

Comment:
We need a name for this form, perhaps `absolute model name`?
```suggestion
	absoluteModelName := fmt.Sprintf("%q (%s)", fmt.Sprintf("%s/%s", cfg.ModelQualifier, cfg.ModelName), modelUUID)
```